README.md

```diff
@@ -121,6 +121,16 @@ different dependencies in your application will be lost. You should also
 set the `isolatedModules` TypeScript option if you plan to ever make use
 of this.
 
+##### logLevel *(string) (default=info)*
+
+Can be `info`, `warn` or `error` which limits the log output to the specified log level.
+Beware of the fact that errors are written to stderr and everything else is written to stdout.
```

Comment:
Could we specify details about `logInfoToStdOut` before `logLevel` in the documentation?  Also, instead of:

> errors are written to `stderr` and everything else is written to `stdout`

could we say:

> errors are written to `stderr` and everything else is written to `stderr` (or `stdout` if `logInfoToStdOut` is `true`)